.circleci/config.yml

```diff
@@ -15,6 +15,7 @@ jobs:
       - checkout
       - run: # install and activate virtual environment with pip
           command: |
+            rm Dockerfile
```

Comment:
there is this nice "feature" that detects a dockerfile and makes the assumption that you're trying to use a custom runtime environment in GAE. that is not actually the case here.

for building the docker image, the `.dockerignore` is sufficient for getting past this because the Dockerfile isn't used in the image itself. for circle, we need to actually remove the Dockerfile. fortunately it has no use in either of these cases.